crates/sdk/Cargo.toml

```diff
@@ -18,14 +18,14 @@ path = "bin/main.rs"
 [features]
 default = ["native-tls", "ndc-test"]
 
-native-tls = ["ndc-client/native-tls", "ndc-test/native-tls", "reqwest/native-tls"]
-rustls = ["ndc-client/rustls", "ndc-test/rustls", "reqwest/rustls"]
+native-tls = ["reqwest/native-tls"]
```

Comment:
Note: the ndc-test feature was added back [here](https://github.com/hasura/ndc-spec/pull/135), but for now, there's no way of using rustls for the embedded `ndc-test`.